Assets/Scripts/FadeToButton.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FadeToButton : MonoBehaviour
+{
+    BlackFade fader => GameObject.Find("FadingCanvas").GetComponent<BlackFade>();
```

Comment:
1. Make sure to always specify if its public/private. It defaults to private when nothing is there, but it's good practice to always put it. Makes the code easier to read and understand. In this case I'd make it a private. If you do decide to make it private, remember it has to have a leading _ in the name. In VSCode you can hit F2 to change variable names and it'll change it for the whole file.
Also if it's private, you can have it show up on the Inspector window in unity if on the line above you put
`[SerializeField]`

2. In this case it's okay to use Find, but you should change it to what I showed on the Discord just so you know how. It's best to avoid Find as much as possible because what it does is it looks through your whole project, and in theory if this was a gigantic project there's potential slowdown. This is what people mean when they talk about game optimization btw. They fix a million small things and make them a tiny bit more efficient. In this case it's mostly okay because we only do it once and this project is super tiny.